api.planx.uk/modules/gis/service/local_authorities/metadata/base.ts

```diff
@@ -27,25 +27,18 @@ const baseSchema: PlanningConstraintsBaseSchema = {
     "digital-land-datasets": ["central-activities-zone"],
     category: "General policy",
   },
-  listed: {
+  brownfieldSite: {
```

Comment:
I've alphabetised the active datasets while here - apologies it makes for a noisy diff!